src/screens/SplashScreen/index.js

```diff
@@ -48,9 +49,14 @@ const SplashScreen = () => {
     try {
       const id = await getUserId();
       const anonymousUserId = await getAnonymousUserId();
-      setProfileId({
+      const token = TokenStorage.get(ITokenEnum.token);
+      const anonymousToken = TokenStorage.get(ITokenEnum.anonymousToken);
+
+      setAuth({
         anonProfileId: anonymousUserId,
-        profileId: id
+        profileId: id,
+        token,
+        anonymousToken
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The retrieval of `token` and `anonymousToken` from `TokenStorage` and their inclusion in the `setAuth` call is a significant change. It appears to be correctly implemented, assuming that `TokenStorage.get` returns the expected values for `ITokenEnum.token` and `ITokenEnum.anonymousToken`.
```diff
- setProfileId({
+ const token = TokenStorage.get(ITokenEnum.token);
+ const anonymousToken = TokenStorage.get(ITokenEnum.anonymousToken);
+ 
+ setAuth({
    anonProfileId: anonymousUserId,
    profileId: id,
+   token,
+   anonymousToken
});
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->